agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3392,13 +3398,14 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
         master_socket = backhaul_manager_socket;
         master_socket->setPeerMac(backhaul_params.controller_bridge_mac);
 
+        auto db = AgentDB::get();
```

Comment:
generally speaking:
- the `AgentDB::get()` is blocking - so define it as close to the first point of use

- if it was not blocking and don't want to do other operations if fail to lock then ok to put at the beginning of function/scope.

Reply:
I put it here since I plan to add more parameters to the db that will require it and I don't want to move it each time.